src/vs/workbench/contrib/audioCues/browser/audioCueLineFeatureContribution.ts

```diff
@@ -41,7 +42,8 @@ export class AudioCueLineFeatureContribution
 		@IEditorService private readonly editorService: IEditorService,
 		@IInstantiationService private readonly instantiationService: IInstantiationService,
 		@IAudioCueService private readonly audioCueService: IAudioCueService,
```

Comment:
> `@IAudioCueService private readonly audioCueService: IAudioCueService,`

It is very fishy that this contribution now uses both the `audioCueService` and the `accessibleNotificationService`.

And indeed this causes bugs, as the line-feature enablement state is taken from the audio cue service. So if some audio cues are disabled, the corresponding `accessibleNotificationEvent` will not be sent either.